CollAction/Views/Shared/_EmbeddedLayout.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿<!DOCTYPE html>
```

Comment:
Why is this layout necessary? I don't see it being used?

Reply:
This file comes from the Friesland-branch. It's used to display individual project tiles in an iframe. That function is not present in Master though.

Reply:
fixed in d0cb67b